src/scripts/tenantDataMigrations/migrate-existing-tenant-data.js

```
@@ -0,0 +1,1965 @@
+				{ name: 'notification_templates', columns: ['tenant_code', 'organization_code'] },
+				{ name: 'organization_extension', columns: ['tenant_code', 'organization_code'] },
+				{ name: 'report_queries', columns: ['tenant_code', 'organization_code'] },
+				{ name: 'reports', columns: ['tenant_code', 'organization_code'] },
+				{ name: 'role_extensions', columns: ['tenant_code', 'organization_code'] },
+			],
+
+			userExtensions: { name: 'user_extensions', columns: ['tenant_code', 'organization_code'] },
+
+			withEntityTypeId: [{ name: 'entities', columns: ['tenant_code'], entityTypeIdColumn: 'entity_type_id' }],
+
+			withUserId: [
+				{ name: 'sessions', columns: ['tenant_code'], userIdColumn: 'created_by' },
+				{ name: 'feedbacks', columns: ['tenant_code'], userIdColumn: 'user_id' },
+				{ name: 'connection_requests', columns: ['tenant_code'], userIdColumn: 'created_by' },
+				{ name: 'connections', columns: ['tenant_code'], userIdColumn: 'created_by' },
+				{ name: 'resources', columns: ['tenant_code'], userIdColumn: 'created_by' },
+				{ name: 'session_request', columns: ['tenant_code'], userIdColumn: 'created_by' },
+				{ name: 'issues', columns: ['tenant_code', 'organization_code'], userIdColumn: 'user_id' },
+				{ name: 'question_sets', columns: ['tenant_code', 'organization_code'], userIdColumn: 'created_by' },
+				{ name: 'questions', columns: ['tenant_code', 'organization_code'], userIdColumn: 'created_by' },
+			],
+
+			withSessionId: [
+				{ name: 'post_session_details', columns: ['tenant_code'], sessionIdColumn: 'session_id' },
+				{ name: 'session_attendees', columns: ['tenant_code'], sessionIdColumn: 'session_id' },
+			],
+
+			withDefaults: [
+				{ name: 'modules', columns: ['tenant_code'] },
+				{ name: 'report_types', columns: ['tenant_code', 'organization_code'] },
+				{ name: 'report_role_mapping', columns: ['tenant_code', 'organization_code'] },
+			],
+		}
+	}
+
+	async loadLookupData() {
```

CSV must be mandatory and must not default to sample data.
If CSV is missing, the script should abort instead of falling back to defaults.